File: src/main/java/mygame/editor/render/TreeItemPath.java
package mygame.editor.render;

import javafx.scene.control.cell.TextFieldTreeCell;
import mygame.editor.model.TreeFileHolder;


public class TreeItemPath extends TextFieldTreeCell<TreeFileHolder>{
    @Override
    public void updateItem(TreeFileHolder item, boolean empty) {
        super.updateItem(item, empty);
        if(item!=null){
            setText(item.getName());
        }
    }
}

File: src/main/java/mygame/editor/ui/TreeItemPath.kt
package mygame.editor.ui

import javafx.scene.control.TreeItem
import javafx.scene.control.cell.TextFieldTreeCell
import java.nio.file.Path


class TreeItemPath : TextFieldTreeCell<Path>(){
    override fun updateItem(item: Path?, empty: Boolean) {
        super.updateItem(item, empty)
        text = item?.toFile()?.name
    }
}
